orchestrator/careplancontributor/taskengine/validation_test.go

```diff
@@ -0,0 +1,88 @@
+package taskengine
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/SanteonNL/orca/orchestrator/careplancontributor/taskengine/testdata"
+	"github.com/stretchr/testify/require"
+	"github.com/zorgbijjou/golang-fhir-models/fhir-models/fhir"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func TestFHIRValidatorCLIQuestionnaireResponseValidator_Validate(t *testing.T) {
+	_, err := exec.LookPath("java")
+	if err != nil {
+		t.Skip("not running FHIR validator test, no Java installed")
+	}
+	err = downloadFHIRValidatorCLI()
+	require.NoError(t, err)
+	ctx := context.Background()
+
+	validator := FHIRValidatorAPIQuestionnaireResponseValidator{URL: "http://localhost:3500/validate", client: &http.Client{}}
```

Comment:
the JAR is never started, maybe change this test to use testcontainers. Search for `SetupHAPI` in the codebase, for an example with HAPI.